tests/test_storage.py

```diff
@@ -216,3 +216,17 @@ def test_get_column_concatenated_bytes_files_path(self, mock_get_config: mock.Mo
         # dataset_name, raise ValueError
         with self.assertRaises(ValueError):
             S3PathFactory().get_column_concatenated_bytes_files_path()
+
+        # Test the remove s3 prefix option in the get_column_concatenated_bytes_files_path function
+        self.assertEqual(
+            S3PathFactory().get_column_concatenated_bytes_files_path(dataset_name=dataset_name, s3_prefix=False),
+            f"dummy_bucket/wicker/{dataset_name}/__COLUMN_CONCATENATED_FILES__",
+        )
+
+        # Test when the s3 prefix remove bool is not passed the prefix isn't eliminated.
+        self.assertEqual(
+            S3PathFactory(prefix_replace_path="/test_mount_path").get_column_concatenated_bytes_files_path(
+                dataset_name=dataset_name
+            ),
+            f"s3://dummy_bucket/wicker/{dataset_name}/__COLUMN_CONCATENATED_FILES__",
```

Comment:
@pickles-bread-and-butter let me get one test where `s3_prefix=True` and `prefix_replace_path="/test_mount_path"`

Reply:
Yep that's what this one does, the s3_prefix=True by default. I'll pass it explicitly though.